--- /usr/local/lib/python3.6/dist-packages/keras/engine/saving.py	2021-01-03 22:38:22.798743431 +0000
+++ saving.py	2021-01-03 22:37:32.485372564 +0000
@@ -718,14 +718,15 @@
     # Returns
         data: Attributes data.
     """
+    data = []
     if name in group.attrs:
-        data = [n.decode('utf8') for n in group.attrs[name]]
+        for n in group.attrs[name]:
+            data.append(n if isinstance(n, str) else n.decode('utf8'))
     else:
-        data = []
         chunk_id = 0
         while ('%s%d' % (name, chunk_id)) in group.attrs:
-            data.extend([n.decode('utf8')
-                         for n in group.attrs['%s%d' % (name, chunk_id)]])
+            for n in group.attrs['%s%d' % (name, chunk_id)]:
+                data.append(n if isinstance(n, str) else n.decode('utf8'))
             chunk_id += 1
     return data
 
@@ -1180,11 +1181,11 @@
             and weights file.
     """
     if 'keras_version' in f.attrs:
-        original_keras_version = f.attrs['keras_version'].decode('utf8')
+        original_keras_version = f.attrs['keras_version']
     else:
         original_keras_version = '1'
     if 'backend' in f.attrs:
-        original_backend = f.attrs['backend'].decode('utf8')
+        original_backend = f.attrs['backend']
     else:
         original_backend = None
 
@@ -1259,11 +1260,11 @@
             and weights file and skip_mismatch=False.
     """
     if 'keras_version' in f.attrs:
-        original_keras_version = f.attrs['keras_version'].decode('utf8')
+        original_keras_version = f.attrs['keras_version']
     else:
         original_keras_version = '1'
     if 'backend' in f.attrs:
-        original_backend = f.attrs['backend'].decode('utf8')
+        original_backend = f.attrs['backend']
     else:
         original_backend = None
 
